src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -84,9 +82,8 @@ const perfherderFormatter = series => {
   const data = {
     datasets: selectFrom(combinedSeries)
       .enumerate()
-      .map(({ data, ...row }, index) => ({
+      .map(({ data, ...row }) => ({
```

Comment:
As above, why is the `index` parameter not needed anymore?

Reply:
`index` was only needed in the line below: `..generateDatasetStyle(SETTINGS.colors[index])`.  That line is not needed because `data.datasets` is headed toward `<ChartJsWrapper>`, which requires less default styling.  The compoenent needs less styling because it generates its own defaults https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/415/files#diff-93b7b23a8e3bd2bd3d6737147526596cR90